la_funding_analysis/utils/plotters.py

```diff
@@ -0,0 +1,129 @@
+# File: utils/plotters.py
+"""Functions to assist with plotting.
+"""
+
+import numpy as np
+import pandas as pd
+import matplotlib.pyplot as plt
+
+
+def rand_jitter(arr):
+    # Function to add small variation to values in a list.
+    stdev = 0.01 * (max(arr) - min(arr))
+    return arr + np.random.randn(len(arr)) * stdev
+
+
+def jitter(
+    x,
+    y,
+    s=20,
+    c="b",
+    marker="o",
+    cmap=None,
+    norm=None,
+    vmin=None,
+    vmax=None,
+    alpha=None,
+    linewidths=None,
+    verts=None,
+    hold=None,
+    **kwargs
+):
+    # Function to make a jitter plot, with jitter only in the x direction.
+    return plt.scatter(
+        rand_jitter(x),
+        y,
+        s=s,
+        c=c,
+        marker=marker,
+        cmap=cmap,
+        norm=norm,
+        vmin=vmin,
+        vmax=vmax,
+        alpha=alpha,
+        linewidths=linewidths,
+        **kwargs
+    )
+
+
+def strip_plot(data, value, ymin, ymax, graph_ylabel, graph_title, filename):
```

Comment:
This plot is for specifically plotting grants, but that only becomes obvious when you read the comments or code. Might be worth having a more descriptive name

Reply:
Have changed all plotter names to be more descriptive